ui/src/app/tags/components/DefinitionField/DefinitionField.tsx

```diff
@@ -35,12 +35,13 @@ const getDefinitionError = (
   definitionErrorId: string
 ) => {
   if (errors.definition?.includes(INVALID_XPATH_ERROR)) {
-    // TODO: Add the link to the docs:
-    // https://github.com/canonical-web-and-design/app-tribe/issues/748
     return (
       <span id={definitionErrorId}>
         The definition is an invalid XPath expression. See our{" "}
-        <a href="#todo">XPath documentation</a> for more examples.
+        <a href="https://maas.io/docs/how-to-work-with-tags#heading--xpath-expressions">
```

Comment:
There is no heading--xpath-expressions id on this page

Reply:
There are no examples of valid and invalid XPath expressions in these docs.

Reply:
Can we skip the content part for now? I created another [ticket](https://github.com/canonical-web-and-design/app-tribe/issues/807) to QA Doc content and dead links.

Reply:
Ok, seems strange to QA something that is broken but as long as the content is published and tested before the release then :+1: